dev-docs/sidebars.js

```diff
@@ -31,7 +31,7 @@ const sidebars = {
     {
       type: "category",
       label: "@excalidraw/excalidraw",
-
+      collapsed: false,
```

Comment:
package docs are currently the main thing to go to docs for, so let's reduce the friction and also make it more visible